containerops/dockyard
# merge server to router/handle

appv1 server side is now merged to  router/handle/module/models!

way to test it:
- add following settings to conf/runtime.conf just as I added to README.md
```
[updateserver]
keymanager = "local://tmp/containerops_keymanager_cache"
storage = "local://tmp/containerops_storage_cache"
```
- compile and run dockyard
- try to push a file
```
./dockyard web --port=1234
curl -d {"this is the test file"} --request PUT localhost:1234/app/v1/tn/tr/tos/tarch/tappname
```
- then we can see files under `/tmp/containerops_storage_cache`

Signed-off-by: liang chenye <liangchenye@huawei.com>
